### PR TITLE
Fix resizing of AboutModal

### PR DESCRIPTION
**Pull request by moores2**
Contributes to #390

Fix the problem with dynamic resizing of the AboutModal not enabling scrolling. Done by removing the ReactResizeDetector component and associated logic.

### CODE REVIEW

File: packages/experimental/src/components/AboutModal/AboutModal.js [hunk 1]
```diff
@@ -33,117 +32,93 @@ export const AboutModal = ({
...
+        titleClassName={`${pkg.prefix}-about-modal-title`}
+      />
+      <ModalBody className={`${pkg.prefix}-about-modal-content`}>
+        <div className={`${pkg.prefix}-about-modal-body-content`}>
+          {content}
+          <div className={`${pkg.prefix}-about-modal-links-container`}>
+            {links &&
+              links.length > 0 &&
+              links.map((link, i) => (
+                <React.Fragment key={i}>
```
[the remark this answers is not in the document]
Comment (moores2) [reply packages/experimental/src/components/AboutModal/AboutModal.js]: Fixed

File: packages/experimental/src/components/AboutModal/AboutModal.js [hunk 2]
```diff
@@ -33,117 +32,93 @@ export const AboutModal = ({
...
+                  {link}
+                  {i !== links.length - 1 && (
+                    <span className={`${pkg.prefix}-about-modal-link-divider`}>
+                      |
+                    </span>
+                  )}
+                </React.Fragment>
+              ))}
+          </div>
+          {legalText ? (
```
[the remark this answers is not in the document]
Comment (moores2) [reply packages/experimental/src/components/AboutModal/AboutModal.js]: Fixed

File: packages/experimental/src/components/AboutModal/AboutModal.js [hunk 3]
```diff
@@ -33,117 +32,93 @@ export const AboutModal = ({
...
+            <p className={`${pkg.prefix}-about-modal-legal-text`}>
+              {legalText}
+            </p>
+          ) : null}
+          {copyrightText ? (
```
[the remark this answers is not in the document]
Comment (moores2) [reply packages/experimental/src/components/AboutModal/AboutModal.js]: Fixed

**Review by davidmenendez — APPROVED** [review]
looks good 👍

**Review by dcwarwick — CHANGES_REQUESTED** [review]
I love it when a defect can be addressed by simplifying and removing code! There's just one glitch I've spotted, when the height becomes extremely small the scroll-fade rises up over the modal header:

image

**Review by dcwarwick — APPROVED** [review]

**Pull request merged**